src/basics/Grid.js

```diff
@@ -113,6 +113,8 @@ const ColumnEl = styled.div`
     css`
       overflow-y: scroll;
       height: 100vh;
```

Comment:
looks like we don't need this old height property anymore, right?

Reply:
I'd like to keep it just in case users visit it via browsers that don't support custom vh

Reply:
ah gotcha. that makes sense